salt/fileserver/gitfs.py

```diff
@@ -50,7 +50,7 @@
 
 PER_REMOTE_OVERRIDES = ('base', 'mountpoint', 'root', 'ssl_verify',
                         'env_whitelist', 'env_blacklist', 'refspecs')
-PER_REMOTE_ONLY = ('name', 'saltenv')
+PER_REMOTE_ONLY = ('formula', 'name', 'saltenv')
```

Comment:
Perhaps a synonym? `world`, `wordwide`, or `universal` (I think I like universal the best?)